dotnetv3/EventBridge/README.md

```diff
@@ -0,0 +1,121 @@
+* [Create a rule](Actions/EventBridgeWrapper.cs#L168) (`PutRule`)
+* [Delete a rule](Actions/EventBridgeWrapper.cs#L435) (`DeleteRule`)
+* [Describe a rule](Actions/EventBridgeWrapper.cs#L35) (`DescribeRule`)
+* [Disable a rule](Actions/EventBridgeWrapper.cs#L71) (`DisableRule`)
+* [Enable a rule](Actions/EventBridgeWrapper.cs#L54) (`EnableRule`)
+* [List rule names for a target](Actions/EventBridgeWrapper.cs#L142) (`ListRuleNamesByTarget`)
+* [List rules](Actions/EventBridgeWrapper.cs#L89) (`ListRules`)
+* [List targets for a rule](Actions/EventBridgeWrapper.cs#L116) (`ListTargetsByRule`)
+* [Remove targets from a rule](Actions/EventBridgeWrapper.cs#L393) (`RemoveTargets`)
+* [Send events](Actions/EventBridgeWrapper.cs#L292) (`PutEvents`)
+
+### Scenarios
+
+Code examples that show you how to accomplish a specific task by calling multiple
+functions within the same service.
+
+* [Get started with rules and targets](Scenarios/EventBridgeScenario.cs) 
+
+## Run the examples
+
+### Prerequisites
+
+For prerequisites, see the [README](../README.md#Prerequisites) in the `dotnetv3` folder.
+
+<!--custom.prerequisites.start-->
+<!--custom.prerequisites.end-->
+
+### Instructions
+
+<!--custom.instructions.start-->
+Before you compile the .NET application, you can optionally set configuration values
+in the settings.json file. These settings include topic and bucket name settings for
+working with rules and targets. Alternatively, add a settings.local.json file with
+your local settings, which will be loaded automatically when the application runs.
+
+After the example compiles, you can run it from the command line. To do so, navigate to
+the folder that contains the .csproj file and run the following command:
+
+```
+dotnet run
+```
+Alternatively, you can run the example from within your IDE.
+<!--custom.instructions.end-->
+
+#### Hello EventBridge
+
+This example shows you how to get started using EventBridge.
+
+
+#### Get started with rules and targets
+
+This example shows you how to do the following:
+
+* Create a rule.
+* Add a target to a rule.
+* Enable and disable rules.
+* List rules and targets.
+* Update rules and targets.
+* Send events.
+* Delete the rule.
+
+<!--custom.scenarios.eventbridge_Scenario_GettingStarted.start-->
+<!--custom.scenarios.eventbridge_Scenario_GettingStarted.end-->
+
+### Tests
+
+⚠ Running tests might result in charges to your AWS account.
+
+To find instructions for running these tests, see the [README](../README.md#Tests)
+in the `dotnetv3` folder.
+
+<!--custom.tests.start-->
+<!--custom.tests.end-->
+
+## Additional resources
+
+* [EventBridge User Guide](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-what-is.html)
+* [EventBridge API Reference](https://docs.aws.amazon.com/eventbridge/latest/APIReference/Welcome.html)
+* [SDK for .NET EventBridge reference](https://docs.aws.amazon.com/sdkfornet/v3/apidocs/items/Eventbridge/NEventbridge.html)
```

Comment:
This link is broken. It's EventBridge (CamelCase):
```
https://docs.aws.amazon.com/sdkfornet/v3/apidocs/items/EventBridge/NEventBridge.html
```

Reply:
Fixed.